### PR TITLE
test: enable clustered index in test

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
enable clustered index in the test

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: tests/tidb_rowid/run.sh [hunk 1]
```diff
@@ -17,6 +17,11 @@
 
 set -eu
 
+# disable cluster index by default
+run_sql 'set @@global.tidb_enable_clustered_index = 0' || echo "tidb does not support cluster index yet, skipped!"
```
Comment (kennytm) [inline tests/tidb_rowid/run.sh:21]: should reset the value back to the original after finish running

**Review by kennytm — COMMENTED** [review]
rest LGTM

File: tests/tidb_rowid/run.sh (hunk 1 above)
Comment (leoppro) [reply tests/tidb_rowid/run.sh:21]: fixed

Comment (glorv) [inline tests/tidb_rowid/run.sh:21]: I think it's better to check the origin setting like https://github.com/pingcap/tidb-lightning/blob/463e7289fcbc412a49e50b307e3b33d30da7f218/tests/row-format-v2/run.sh#L8-L28

BTW, the newest version of `show variables like % tidb_enable_clustered_index%` return ON/OFF

**Comment by kennytm** [comment]
/run-all-tests tidb=release-5.0 tikv=release-5.0 pd=release-5.0 importer=pr/89

**Comment by kennytm** [comment]
/run-all-tests tidb=release-5.0-rc tikv=release-5.0-rc pd=release-5.0-rc importer=pr/89

**Comment by lance6716** [comment]
ping @leoppro

**Comment by lance6716** [comment]
need address comments @leoppro

**Pull request closed without merging**